packages/analytics/dashboard-renderer/src/components/GeoMapRenderer.vue

```diff
@@ -60,6 +61,10 @@ const countryMetricUnit = computed(() => {
   return metricUnits && metricNames && metricUnits[metricNames[0]]
 })
 
+const truncatedMetric = computed(() => {
```

Comment:
We could move this into the geo map component rather than adding the `truncatedMetric` prop

Rather than the truncated metric prop, I'd rather pass the metric through to the geo map component 🤔